scale/job/execution/running/tasks/base_task.py

```diff
@@ -18,15 +18,20 @@ def __init__(self, task_id, job_exe):
         """Constructor
 
         :param task_id: The unique ID of the task
-        :type task_id: str
+        :type task_id: string
         :param job_exe: The job execution, which must be in RUNNING status and have its related node, job, job_type, and
             job_type_rev models populated
         :type job_exe: :class:`job.models.JobExecution`
         """
 
         self._task_id = task_id
-        self._task_name = 'Job Execution %i (%s)' % (job_exe.id, job_exe.get_job_type_name())
-        self._is_running = False
+        self._task_name = '%s %s' % (job_exe.job.job_type.title, job_exe.job.job_type.version)
```

Comment:
No, there are no (and shouldn't be) any dependencies on the task name. It's for display purposes only. The new task name is more helpful and descriptive in a DC/OS environment with other tasks/frameworks running.